runtime/Cargo.toml

```diff
@@ -50,48 +50,96 @@ serde = { workspace = true, features = ["rc"] }
 serde_derive = { workspace = true }
 serde_json = { workspace = true }
 serde_with = { workspace = true }
+solana-account = { workspace = true }
+solana-account-info = { workspace = true }
 solana-accounts-db = { workspace = true }
+solana-address-lookup-table-interface = { workspace = true }
 solana-bpf-loader-program = { workspace = true }
 solana-bucket-map = { workspace = true }
 solana-builtins = { workspace = true }
+solana-client-traits = { workspace = true }
 solana-clock = { workspace = true }
+solana-commitment-config = { workspace = true }
 solana-compute-budget = { workspace = true }
 solana-compute-budget-instruction = { workspace = true }
+solana-compute-budget-interface = { workspace = true }
 solana-cost-model = { workspace = true }
+solana-cpi = { workspace = true }
+solana-ed25519-program = { workspace = true }
+solana-epoch-info = { workspace = true }
+solana-epoch-rewards-hasher = { workspace = true }
+solana-epoch-schedule = { workspace = true }
+solana-feature-gate-interface = { workspace = true }
 solana-fee = { workspace = true }
+solana-fee-calculator = { workspace = true }
+solana-fee-structure = { workspace = true }
 solana-frozen-abi = { workspace = true, optional = true, features = [
     "frozen-abi",
 ] }
 solana-frozen-abi-macro = { workspace = true, optional = true, features = [
     "frozen-abi",
 ] }
+solana-genesis-config = { workspace = true }
+solana-hard-forks = { workspace = true }
+solana-hash = { workspace = true }
+solana-inflation = { workspace = true }
+solana-instruction = { workspace = true }
+solana-keypair = { workspace = true }
 solana-lattice-hash = { workspace = true }
+solana-loader-v3-interface = { workspace = true }
 solana-loader-v4-interface = { workspace = true, features = ["serde"] }
 solana-measure = { workspace = true }
+solana-message = { workspace = true }
 solana-metrics = { workspace = true }
+solana-native-token = { workspace = true }
 solana-nohash-hasher = { workspace = true }
+solana-nonce = { workspace = true }
 solana-nonce-account = { workspace = true }
+solana-packet = { workspace = true }
 solana-perf = { workspace = true }
+solana-poh-config = { workspace = true }
 solana-precompile-error = { workspace = true }
 solana-program = { workspace = true }
 solana-program-runtime = { workspace = true, features = ["metrics"] }
 solana-pubkey = { workspace = true }
 solana-rayon-threadlimit = { workspace = true }
+solana-rent = { workspace = true }
+solana-rent-collector = { workspace = true }
+solana-rent-debits = { workspace = true }
+solana-reward-info = { workspace = true }
 solana-runtime-transaction = { workspace = true }
 solana-sdk = { workspace = true }
+solana-sdk-ids = { workspace = true }
```

Comment:
Same as above, still some users of `solana-sdk` ?

Reply:
https://github.com/anza-xyz/agave/pull/6113#discussion_r2078652188